extract.py

```diff
@@ -0,0 +1,107 @@
+from collections import defaultdict
+import difflib
+import json
+from typing import List, Tuple
+
+from dulwich.diff_tree import TreeChange
+from dulwich.repo import Repo
+from dulwich.walk import WalkEntry
+
+
+def calc_diff(repo: Repo, change: TreeChange) -> Tuple[int, int]:
+    """
+    Get amount of added and deleted lines
+
+    :param repo: repository with change
+    :param change: single change between two trees
+    :return: number of lines added/deleted
+    """
+    d = difflib.Differ()
+    diffs = d.compare(repo.get_object(change.old.sha).data.decode().splitlines(),
```

Comment:
you can make lambda like `into_lines = lambda sha: repo.get_object(sha).data.decode().splitlines()` - so logic is not duplicated